src/shaders/fill_extrusion_depth.fragment.glsl

```diff
@@ -1,5 +1,8 @@
 varying highp float v_depth;
 
 void main() {
+#ifdef DEPTH_TEXTURE
```

Comment:
maybe `#ifndef` makes more sense here?

Reply:
I agree, but this is a workaround for an issue in the Metal shader generator not producing correct output with `#ifndef`. Same applies to the conditional declaration of the varying property.

Reply:
@endanke confirmed offline that the issue isn't fixed yet. I can add a comment here stating why the code is like this and when it can be updated.

Reply:
Checked it now and we can fix the generator in gl-native in @aleksigron's PR :)